### PR TITLE
Query id invalid exception

### PR DESCRIPTION
**Pull request by karb0f0s**
- add QueryIdInvalidException

### CODE REVIEW

File: src/Telegram.Bot/Exceptions/BadRequestExceptions/QueryIdInvalidException.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿// ReSharper disable once CheckNamespace
```
Comment (pouladpld) [inline src/Telegram.Bot/Exceptions/BadRequestExceptions/QueryIdInvalidException.cs]: File should be in `Invalid Parameter` directory.

File: src/Telegram.Bot/Exceptions/BadRequestExceptions/QueryIdInvalidException.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+namespace Telegram.Bot.Exceptions
+{
+    /// <summary>
+    /// The exception that is thrown when Query Id is invalid or AnswerInlineQueryAsync 
+    /// called with 10 second delay
+    /// </summary>
+    public class QueryIdInvalidException : BadRequestException
```
Comment (pouladpld) [inline src/Telegram.Bot/Exceptions/BadRequestExceptions/QueryIdInvalidException.cs]: Class could inherit `InvalidParameterException`.

Comment (pouladpld) [inline src/Telegram.Bot/Exceptions/BadRequestExceptions/QueryIdInvalidException.cs]: Class name could be `InvalidQueryIdException`.

Comment (pouladpld) [reply src/Telegram.Bot/Exceptions/BadRequestExceptions/QueryIdInvalidException.cs]: Pass `inline_query_id` to base ctor.

File: test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs [hunk 3]
```diff
@@ -104,6 +105,40 @@ public ApiExceptionsTests(TestsFixture fixture)
...
+            Update queryUpdate = await _fixture.UpdateReceiver.GetInlineQueryUpdateAsync();
+
+            const string resultId = "article:bot-api";
+            InputMessageContentBase inputMessageContent =
+                new InputTextMessageContent("https://core.telegram.org/bots/api");
+
+            InlineQueryResultBase[] results =
+            {
+                new InlineQueryResultArticle(
+                    id: resultId,
```
Comment (pouladpld) [inline test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs]: value could be set inline.

File: test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs [hunk 4]
```diff
@@ -104,6 +105,40 @@ public ApiExceptionsTests(TestsFixture fixture)
...
+
+            await Task.Delay(10_000);
+
+            BadRequestException e = await Assert.ThrowsAnyAsync<BadRequestException>(() =>
+                BotClient.AnswerInlineQueryAsync(
+                    inlineQueryId: queryUpdate.InlineQuery.Id,
+                    results: results,
+                    cacheTime: 0));
+
+            Assert.IsType<QueryIdInvalidException>(e);
```
Comment (pouladpld) [inline test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs]: Assert on `e.Parameter` value.

File: test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs [hunk 5]
```diff
@@ -104,6 +105,40 @@ public ApiExceptionsTests(TestsFixture fixture)
...
+                    title: "Telegram Bot API",
+                    inputMessageContent: inputMessageContent)
+                {
+                    Description = "The Bot API is an HTTP-based interface created for developers",
+                },
+            };
+
+            await Task.Delay(10_000);
+
+            BadRequestException e = await Assert.ThrowsAnyAsync<BadRequestException>(() =>
```
Comment (pouladpld) [inline test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs]: Throws `InvalidQueryIdException`

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
```
Comment (pouladpld) [inline CHANGELOG.md]: `### Added`

Comment (karb0f0s) [reply CHANGELOG.md]: done

**Pull request closed without merging**